articles/custom_storage_location.md

```diff
@@ -61,73 +101,21 @@ For read-write permissions (you allow Synapse to upload and retrieve files)
             "Principal": { "AWS": "325565585839" }
         },
         {
-            "Action": [ "s3:*Object*", "s3:*MultipartUpload*" ],
+            "Action": [ "s3:GetObject*", "s3:*MultipartUpload*" ],
             "Effect": "Allow",
             "Resource": "arn:aws:s3:::synapse-share.yourcompany.com/*",
             "Principal": { "AWS": "325565585839" }
         }
     ]
 }
-````
-
-
-## **Downloading requester pays data**
-
-For data made available through Synapse that is large enough to incur significant download costs, there is a separate method for downloading:
-
-1. Create a new AWS S3 bucket in the us-east-1 region under your account (see [Creating a download bucket](https://www.synapse.org/#!Synapse:syn3316305/wiki/217751) for details)
-2. Use the s3FileCopy API to copy the files to that new bucket
-3. Download the files from that new bucket using the AWS tools
-4. Don't forget to delete the files from that bucket to avoid paying the associated storage costs
-
-
-
-## **External S3 buckets**
-
-## Steps for adding an external S3 bucket
-
-### Share your bucket with Synapse
-First step is to allow Synapse to access your s3 bucket. You can choose to allow read-only access or read-write access. Either way, you have to make sure that objects that are referenced or managed by Synapse are not modified or deleted.
-
-#### 1. Go to **[Amazon Web Services S3 console](https://console.aws.amazon.com/s3)** and login under your account
-
-____
-
-#### 2. Create a new bucket under your account
-Click **Create Bucket**
-
-<img src="/assets/images/create-bucket-button.jpg">
-
-And for the bucket name, use a unique name that ends in your company domain. For example synapse-share.yourcompany.com.
-For region, _you must select the US Standard region_.
+{% endhighlight %}
 
-<img src="/assets/images/create-bucket-dialog.jpg">
+<br/>
 
-____
+#### Read-write permissions
+For read-write permissions (you allow Synapse to upload and retrieve files):
 
```

Comment:
If you don't want to allow authorized  Synapse users to upload data to your bucket but only want to allow read access you can change ....